tutorials/dataframe/df007_snapshot.C

```diff
@@ -36,14 +36,14 @@ int df007_snapshot()
    auto treeName = "myTree";
    fill_tree(treeName, fileName);
 
-   // We read the tree from the file and create a RDataFrame.
+   // We read the tree from the file and create a RDataFrame
    ROOT::RDataFrame d(treeName, fileName);
 
    // ## Select entries
    // We now select some entries in the dataset
    auto d_cut = d.Filter("b1 % 2 == 0");
    // ## Enrich the dataset
-   // Build some temporary columns: we'll write them out
+   // Build some temporary columns: We'll write them out
```

Comment:
as I commented in some other PR, English does not use caps after `:`

Reply:
Sorry, I missed that. I fixed it now.

Reply:
thank you. note that is comment is applicable in few other places as well